x-pack/plugins/cases/docs/openapi/components/schemas/update_case_connector_cases_webhook.yaml

```diff
@@ -0,0 +1,39 @@
+title: Update case request with a Cases Webhook connector
```

Comment:
Just wondering if we could do the same thing for these files, have one single `update_case_connector` and then have the `oneOf` property in here instead of creating an update file per connector. Most of the file is the same expect the `properties.connector`.

Reply:
Good idea!  I've made that work via c83d4276d2187a4fb92b39a09bbf7cfef8cd51af and 4ac43e0ff7a1f94d8f01144908d76854bd02090f